README.md

```diff
@@ -25,6 +25,14 @@ This layer depends on:
 
 Follow the usual steps to setup OpenEmbedded and bitbake.
 
+Steps: 
```

Comment:
Can you change this to "One option to simplify setup is to use poky, these are some example steps to do this:"? Not everyone (including me) uses poky.